content/source/docs/enterprise/install/active-active.html.md

```diff
@@ -404,15 +402,15 @@ locals {
 
 ### Step 3: Connect to External Redis
 
-Once you include the above configuration options in your `settings.json` file, connect a single node to your newly provisioned Redis service. Before scaling to two nodes, ensure your new service functions as expected by testing on a single node.
+Once you are prepared to include the modified configuration options in your configuration files, you must connect a single node to your newly provisioned Redis service by rebuilding your node instance with the new settings.
 
-#### Reprovision Instance
+#### Re-provision Terraform Enterprise Instance
 
-You need to terminate the existing instance by scaling down to zero. Once you terminate the instance, you can scale back up to one instance using your latest config templates. 
+Terminate the existing instance by scaling down to zero. Once terminated, you can scale back up to one instance using your revised configuration. 
 
 #### Wait for Terraform Enterprise to Install
 
-It can take up to 15 minutes for the application to respond as healthy. You can monitor the status of the install by watching your Auto Scaling Group in your cloud’s web console. Alternatively, you can SSH onto the instance and run the following command to monitor the installation directly:
+It can take up to 15 minutes for the node instance to provision and the TFE application to be installed and respond as healthy. You can monitor the status of the node instance provisioning by watching your Auto Scaling Group in your cloud’s web console. To confirm the successful implementation of the TFE application you can SSH onto the instance and run the following command to monitor the installation directly:
```

Comment:
```suggestion
It can take up to 15 minutes for the node to provision and the TFE application to be installed and respond as healthy. You can monitor the status of the node instance provisioning by watching your Auto Scaling Group in your cloud’s web console. To confirm the successful implementation of the TFE application you can SSH onto the instance and run the following command to monitor the installation directly:
```